tests/tensorflow/nightly/targets.jsonnet

```diff
@@ -27,6 +27,7 @@ local shapemask = import 'shapemask.libsonnet';
 local transformer_translate = import 'transformer-translate.libsonnet';
 local vision_coco = import 'vision-coco.libsonnet';
 local vision_imagenet = import 'vision-imagenet.libsonnet';
+local vision_coco = import 'vision-coco.libsonnet';
```

Comment:
This is probably leftover from a merge. `vision_coco` is already defined on line 28